app/models/knight.rb

```diff
@@ -1,2 +1,8 @@
 class Knight < Piece
+  def valid_move?(row_destination, col_destination)
+    row_diff = (self.row - row_destination).abs
+    col_diff = (self.column - col_destination).abs
+
+    (row_diff / col_diff) == 2 || (col_diff / row_diff) == 2 ? true : false
```

Comment:
This looks like a good approach, however larger jumps will also trigger as the correct ratio (i.e. moving up 4 and over 2), and additionally Ruby drops precision on integer arithmetic, so any remainders of those divisions will get dropped (up 5 over 2 will still be 2):

```ruby
4 / 2 == 2 #=> true
5 / 2 == 2 #=> true
```

You'll likely have to compare all the values directly - you can actually compare an array to an array to make it simple:

```ruby
[1, 2] == [row_diff, col_diff] || #...
```

Also, one small style bit, if the results of an if are true and false, the if can be removed entirely and the condition can be used directly:

```ruby
def verbose_one
  if row_diff == 2
    true
  else
    false
  end
end

def verbose_two
  row_diff == 2 ? true : false
end

def simple
  row_diff == 2
end
```

```ruby
def other_verbose
  result = something == 2 ? true : false
  # ...
end

def other_simple
  result = something == 2
  # ...
end
```

Looking good!  🎉